src/users/types.ts

```diff
@@ -85,12 +85,8 @@ export type UserReadFull = UserReadObscured & {
   enable_module_project_timetracking: boolean;
   enable_module_leave_management: boolean;
   enable_module_shift_planning: boolean;
-  enable_module_timetac_health: boolean;
   request_substitute_user_id: number;
   time_tracking_ex_post_earliest_working_time: string;
-  health_rule_id: number;
-  health_rule_id_valid_from: string;
-  health_checker: boolean;
```

Comment:
Not sure if we want to remove the fields from the user, I'd probably only go for the healthData/healthRules endpoints.

@sergej-tschak , what do you think?

Reply:
I think we can do it. they should not be used anywhere else as we remove health API too

Reply:
aight :+1: